samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties

```diff
@@ -247,7 +247,7 @@ select_the_feedback=Select the Feedback Components to be displayed to the studen
 selected=Selected
 selection_level_feed=Selection-level Feedback
 time_expired1=Time Expired!
-time_expired2=The time allowed for this assessment has expired. Your answers have been automatically submitted.
+time_expired2=The time allowed for this assessment has expired. Your answers have been automatically submitted in the next few minutes.
```

Comment:
Hey Brian,

If the student exists the exam a few seconds before times up, say 5 to 15,
and then re enters it a few seconds after tine expired, they are able to
continue even if the timer claims 'Times up'. Give it a try following the
testing steps on the JIRA, its all explained there.
On Jun 6, 2016 8:35 AM, "Brian Jones" <notifications@github.com> wrote:

> In
> samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties
> <https://github.com/sakaiproject/sakai/pull/2755#discussion_r65882011>:
>
> > @@ -247,7 +247,7 @@ select_the_feedback=Select the Feedback Components to be displayed to the studen
> >  selected=Selected
> >  selection_level_feed=Selection-level Feedback
> >  time_expired1=Time Expired!
> > -time_expired2=The time allowed for this assessment has expired. Your answers have been automatically submitted.
> > +time_expired2=The time allowed for this assessment has expired. Your answers have been automatically submitted in the next few minutes.
>
> I'm curious, when does this actually happen? "in the next few minutes" is
> a bit ambiguous, I can see students asking questions about this.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sakaiproject/sakai/pull/2755/files/be7b76fbc2dad2497f949c6dda1ea2ea6f6d12e9#r65882011>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AEnF-O_Cw0YzL7Gs9on74t6OApHV44wgks5qJBQGgaJpZM4It5Ue>
> .
>